docs/development/development_guide_de.md

```diff
@@ -13,7 +13,9 @@
 5. Führen Sie `make install` aus
 6. Exportieren Sie Ihre CES-Instanz-Zugangsdaten, damit der Operator sie verwenden kann
     - `export CES_REGISTRY_USER=instanceId && export CES_REGISTRY_PASS='instanceSecret'`
-7. Führen Sie `make run` aus, um den dogu-Operator lokal auszuführen
```

Comment:
Small annotation: It is easier to edit markdown enumeration when using only 

```
1. Item 1 
1. Item 2
```

instead of 

```
1. Item 1
2. Item 2
```